build.gradle

```diff
@@ -158,8 +161,8 @@ dependencies {
     compileOnly gradleApi()
     compileOnly localGroovy()
     // Don't enable this on the repository because this artifact may include bytecode incompatibility.
-//    compileOnly "com.android.tools.build:gradle:$androidGradleVersion"
-    implementation 'org.jetbrains:annotations:23.1.0'
+    //    compileOnly "com.android.tools.build:gradle:$androidGradleVersion"
+    implementation 'org.jetbrains:annotations:24.0.1'
```

Comment:
Now is the time to unify the usage of annotations like nullability because it requires huge format changes again. We are gonna use jetbrains annotations instead of javax and Guava.